contracts/test/KeepRegistryStub.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity 0.8.9;
+
+import "./IKeepRegistry.sol";
+
+contract KeepRegistryStub is IKeepRegistry {
+    event OperatorContractApproved(address operatorContract);
+
+    address public registryKeeper;
```

Comment:
Do we need this role in the stub at all? We're not checking the sender in `approveOperatorContract`.

Reply:
The `IKeepRegistry` interface requires it.